app-modules/engagement/src/Notifications/EngagementFailedNotification.php

```diff
@@ -72,7 +72,7 @@ public function toDatabase(object $notifiable): array
         return FilamentNotification::make()
             ->danger()
             ->title('Engagement Delivery Failed')
-            ->body("Your engagement failed to be delivered to {$this->engagement->recipient->display_name}.")
+            ->body("Your engagement email failed to be delivered to {$this->engagement->recipient->display_name}.")
```

Comment:
I believe this failed message is currently shared between email and sms right? So we should check which type the engagement is here and say "sms" or "email" respectively.